src/src/rx_main.cpp

```diff
@@ -1005,12 +999,6 @@ bool ICACHE_RAM_ATTR ProcessRFPacket(SX12xxDriverCommon::rx_status const status)
         return false;
     }
 
-    // don't use telemetry packets for PFD calculation since TX does not send such data and tlm frames from other rx are not in sync
```

Comment:
Is this comment no longer relevant?

Reply:
Never really was thinking about it.  Packets will get canned by the CRC.

Reply:
What about the case where the user has 2 RXes, one configured to send telemetry? Will the secondary one receive telemetry from the sending RX?

Reply:
Test as is and the link is fine.  But this is only because PFD is not incremented much once connected, and PACKET_TYPE_TLM packet contents arent processed unless using Airport.

It still isnt great to leave it in this state.  So Ive pushed a commit that places the SX12XX in FS or STANBY modes when tx is 'aborted'.  Either due to ForceTlmOff or LBT.  Now the SX12XX isnt sitting in receiving mode when it is not meant to be.